apis/server/container_bridge.go

```diff
@@ -173,6 +178,7 @@ func (s *Server) getContainers(ctx context.Context, rw http.ResponseWriter, req
 			Labels:          c.Config.Labels,
 			HostConfig:      c.HostConfig,
 			NetworkSettings: netSettings,
+			Mounts:          mounts,
```

Comment:
FWIW，in terms of compatibility, I compared pouchcontainer's returns struct and 

> moby's(https://github.com/moby/moby/blob/master/api/swagger.yaml#L3434-L3501), and found that we still miss:
```
Ports
State
SizeRw
SizeRootfs
```

Reply:
There are no effect at the moment. But it is better to add missing filed in returns struct. @allencloud